homeassistant/components/ohme/__init__.py

```diff
@@ -54,6 +55,8 @@ async def async_setup_entry(hass: HomeAssistant, entry: OhmeConfigEntry) -> bool
 
     entry.runtime_data = OhmeRuntimeData(*coordinators)
 
+    async_setup_services(hass)
```

Comment:
We should setup the services in `async_setup`. This way adding services is not relying on a working config entry to be added. (this is also a rule you can set to done in the quality scale)